### PR TITLE
#WP-21, #WP-24 - Display order number instead of order id

### PR DESCRIPTION
**Pull request by rcdtroly**
### What:
- #WP-21 Display order number instead of order id
- #WP-24 default troly checkout options not correctly displayed

### What:
- Applied minor changes

### Refs:
- https://app.asana.com/0/939825968139797/1136667388142622
- https://app.asana.com/0/939825968139797/1136667388142626

### CODE REVIEW

File: wp99234.php [hunk 1]
```diff
@@ -1196,8 +1195,16 @@ function update_order_meta_birthday( $order_id ) {
 add_action('woocommerce_admin_order_data_after_order_details', 'show_custom_subs_fields', 10, 1 );
 function show_custom_subs_fields( $order ) {
 	$subs_id = get_post_meta( $order->get_id(), 'subs_id', true );
+    $subs_order_no = get_post_meta( $order->get_id(), 'subs_order_no', true );
```
Comment (atifempireone) [inline wp99234.php]: Indentaition is broken here

File: wp99234.php [hunk 2]
```diff
@@ -1196,8 +1195,16 @@ function update_order_meta_birthday( $order_id ) {
...
 	echo '<h3>&nbsp;</h3><h3>Troly Information</h3>';
-	echo '<p><strong>Troly Order ID: </strong>' . $subs_id . '</p>';
+
+    // Show Order no
+    if ($subs_order_no) {
+        echo '<p><strong>Troly Order No: </strong>' . $subs_order_no . '</p>';
+    } else {
+        echo '<p><strong>Troly Order ID: </strong>' . $subs_id . '</p>';
```
Comment (atifempireone) [inline wp99234.php]: when would we not have  subs order no? can we remove this if else?

Comment (rcdtroly) [reply wp99234.php]: for previous orders, we did not store the `$order->number` before

File: wp99234.php (hunk 1 above)
Comment (rcdtroly) [reply wp99234.php]: Fixed

**Pull request merged**